datadog_checks_dev/datadog_checks/dev/tooling/commands/release/tag.py

```diff
@@ -48,6 +48,10 @@ def tag(check, version, push, dry_run):
         if not version:
             version = get_version_string(check)
 
+        if version == '0.0.1':
```

Comment:
Let's comment slightly better on why this is needed. Also, it's not clear why 0.0.1 is not released yet, but is 0.0.2 released?

Reply:
0.0.1 is the initial pre-release version that is generated from the integrations template. 0.0.2 would be a possible release version, since updating to any version > 0.0.1 would require a tag.*.link file to be updated (and the failing pipeline issue would be avoided).

Reply:
Please add this as a comment, thanks!